src/style.css

```diff
@@ -1,3 +1,33 @@
+/* stylelint-disable font-family-name-quotes */
 body {
+  background-color: #858585;
+  font-family: 'Roboto', sans-serif;
   margin: 0;
 }
+
+.nav {
+  justify-content: flex-end;
+  font-style: 15px;
```

Comment:
The `font-style` property is used for setting the font style to normal, italic, or oblique. If you want to set the font size, you should use `font-size` instead.